### DISCUSSION TITLE
Environments / Wait Timers - Ability to skip waiting

### DISCUSSION THREAD

**Discussion by Moser-ss** [community/community#14976] [Actions] [answered]
Hey,

I would like to bring here a topic that I already found in the [Community Forum](https://github.community/t/environments-wait-timers-ability-to-skip-waiting/193175).
The environment has the capability to be protected with a wait timer, but sometimes it will be necessary to skip the wait and deploy faster. For example during an incident.
It will be nice to have a button that allows a power user (like repo admin or code owner) to  skip the wait time

**Comment by yoriiis** [comment, marked as answer]
As an administrator, with the following option enabled, you can start deploying without waiting.

```txt
Allow administrators to bypass configured protection rules
```

Doc: https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#allow-administrators-to-bypass-configured-protection-rules

**Reply by kinsey-roberts-milliman** [reply]
How? I'm in the admin group, but I'm given no option to skip the wait, just this timer countdown.
<img width="505" height="57" alt="image" src="https://github.com/user-attachments/assets/219dac55-27b3-41c3-b43a-b2a31f166c4e" />

**Reply by yoriiis** [reply]
The checkbox below must be checked in the environment settings to enable bypassing. Details are available in the link to the documentation above.

<img width="432" height="33" alt="image" src="https://github.com/user-attachments/assets/7f5573f4-43c0-4726-9d9e-72acefd1f148" />

**Reply by kinsey-roberts-milliman** [reply]
Yes, that checkbox is checked. Like your response, the linked directions do not show how to skip the timer. Please show a screenshot of the UI element with which I must interact to skip the "Waiting for time" screen.

**Reply by yoriiis** [reply]
To ignore the timer when the box is checked, go to the page for the job in question (GitHub Actions summary). At the bottom, there is a section called `Deployment protection rules` with a button labelled `Start all waiting jobs`.

<img width="908" height="197" alt="image" src="https://github.com/user-attachments/assets/33ceda4a-df29-4bab-b8be-dc78e987fdf6" />

Then select the environment to deploy and click on the button labelled `I understand the consequences, start deploying`.

<img width="641" height="410" alt="image" src="https://github.com/user-attachments/assets/91d371ec-d67d-4465-889b-cbf0bb26c06c" />

**Reply by kinsey-roberts-milliman** [reply]
Thank you so much!! I hadn't thought to look at the Summary page.
